src/Microsoft.TestPlatform.Extensions.HtmlLogger/Html.xslt

```diff
@@ -11,18 +11,7 @@
         <h1>Test run details</h1>
         <xsl:apply-templates select ="/tp:TestRunDetails"/>
       </body>
-      <script language="javascript">
-        function ToggleClass(id) {
-        var elem = document.getElementById(id);
-        if (elem.style.display == "none") {
-        elem.style.display = "block";
-        }
-        else {
-        elem.style.display = "none";
-        }
-        }
-      </script>
-      <style>
+        <style>
```

Comment:
Fix indentation

Reply:
This PR is merged, if this bothers you, please start a new PR with the fix.